contracts/badger-ren/strategies/CurveSwapStrategy.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.6.8;
+
+import "deps/@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import "deps/@openzeppelin/contracts-upgradeable/utils/ReentrancyGuardUpgradeable.sol";
+import "deps/@openzeppelin/contracts-upgradeable/math/SafeMathUpgradeable.sol";
+import "deps/@openzeppelin/contracts-upgradeable/token/ERC20/SafeERC20Upgradeable.sol";
+import "deps/@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+
+import "interfaces/curve/ICurveExchange.sol";
+
+contract CurveSwapStrategy is AccessControlUpgradeable, ReentrancyGuardUpgradeable {
```

Comment:
Should you consider creating interface for all future swap strategies to follow coherent pattern?

Reply:
Ah yea I have an `ISwapStrategy` already, I can make it inherit from it.